### PR TITLE
HPCC-18703 Create Username Only Login Screen

### PR DESCRIPTION
**Pull request by kunalaswani**
Create username only login HTML page.

Signed-off-by: Kunal Aswani <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@miguelvazq please review

File: esp/src/eclwatch/templates/GetUserName.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            });
+        </script>
+    </head>
+    <body>
+        <div id="container" class="container visible">
+            <div class="formContainer">
+                <img src="../img/Loginlogo.png" alt="HPCC Systems">
+                <p class="loginStr"></p>
+                <div class="login">
+                    <form method="POST" action="/esp/login">
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/GetUserName.html]: @wangkx Do we still need to call /esp/login when just sending username?

Comment (wangkx) [reply esp/src/eclwatch/templates/GetUserName.html]: Do not have to be 'login'. You may change to different wording.

Comment (miguelvazq) [reply esp/src/eclwatch/templates/GetUserName.html]: Does it have to post to anything? I do not think it needs to call an `action=""` at all?

Comment (wangkx) [reply esp/src/eclwatch/templates/GetUserName.html]: I think we should identify this action (not using now but may use in the future): /esp/setusername.

Comment (miguelvazq) [reply esp/src/eclwatch/templates/GetUserName.html]: We cannot point to an empty action that does not exist. It will simply redirect the page to itself.

Comment (miguelvazq) [reply esp/src/eclwatch/templates/GetUserName.html]: If ESP has logic to know if user is not LDAP enabled and will handle accordingly, this looks fine.

**Comment by kunalaswani** [comment]
@richardkchapman please merge

**Pull request merged**